src/components/AddressForm.tsx

```diff
@@ -126,15 +155,16 @@ function AddressForm({city, country, formID, onAddressChanged, onSubmit, shouldS
         >
             <View>
                 <InputWrapper
+                    // @ts-expect-error TODO: Remove this once InputWrapper (https://github.com/Expensify/App/issues/25109) is migrated to TypeScript.
                     InputComponent={AddressSearch}
                     inputID="addressLine1"
                     label={translate('common.addressLine', {lineNumber: 1})}
-                    onValueChange={(data, key) => {
+                    onValueChange={(data: string, key: string) => {
                         onAddressChanged(data, key);
                         // This enforces the country selector to use the country from address instead of the country from URL
-                        Navigation.setParams({country: undefined});
+                        Navigation.setParams({country: undefined}, '');
```

Comment:
Maybe it will be better to update `Navigation.setParams` typing to make second param optional.

Reply:
updated